src/udisksdaemonutil.c

```diff
@@ -1616,3 +1617,74 @@ udisks_ata_identify_get_word (const guchar *identify_data, guint word_number)
  out:
   return ret;
 }
+
+
+/* Linux kernel version check */
+
+static guint64 detected_linux_version;
```

Comment:
Should be initialized to `0`, right?

Reply:
Nope, static variables are initialized to zero by the compiler. There are some more specifics regarding the use of `static` keyword, see e.g. https://www.geeksforgeeks.org/static-variables-in-c/